public/app/percona/check/CheckPanel.test.tsx

```diff
@@ -37,7 +37,7 @@ jest.mock('@percona/platform-core', () => {
 describe('CheckPanel::', () => {
   beforeEach(() => {
     (useSelector as jest.Mock).mockImplementation((callback) => {
-      return callback({ location: { routeParams: { tab: 'alerts' }, path: '/integrated-alerting/alerts' } });
+      return callback({ perconaUser: { isAuthorized: true }, perconaSettings: { isLoading: false } });
```

Comment:
What does `isAuthorized` actually mean? Maybe you mean authenticated instead of authorized? Since a user can be authenticated but not be authorized to access some resource.

Reply:
This is for when a non-admin user tries to reach a feature page, for example. So, the user is authenticated, but they're not authorised to check some content or call our APIs.

Reply:
Thanks, I was confusing it